package.json

```diff
@@ -22,11 +22,12 @@
   },
   "dependencies": {
     "@nestjs/common": "^7.6.15",
+    "@nestjs/config": "^1.0.0",
     "@nestjs/core": "^7.6.15",
     "@nestjs/platform-fastify": "^7.6.18",
     "@nestjs/swagger": "^4.8.1",
     "@nestjs/typeorm": "^7.1.5",
-    "bcrypt": "^5.0.1",
+    "bcrypt": "^5.0.1", 
```

Comment:
You may remove the extra space after the coma

Reply:
addressed